packages/cojson/src/sync.ts

```diff
@@ -257,66 +237,101 @@ export class SyncManager {
     }
   }
 
-  async sendNewContentIncludingDependencies(id: RawCoID, peer: PeerState) {
-    const coValue = this.local.expectCoValueLoaded(id);
+  async startPeerReconciliation(peer: PeerState) {
+    const coValuesOrderedByDependency: CoValueCore[] = [];
 
-    await Promise.all(
-      coValue
-        .getDependedOnCoValues()
-        .map((id) => this.sendNewContentIncludingDependencies(id, peer)),
-    );
+    const requested = new Set<string>();
```

Comment:
```suggestion
    const gathered = new Set<string>();
```

(`requested` sounds like you're already doing some syncing, when you're just gathering & ordering them)